### ISSUE TITLE
Spell check this regular expression check is too strict

### ISSUE THREAD

**Issue by Luko6** [check-spelling/check-spelling#30]
I'm writing a reusable workflow, and trying to utilize the `config` parameter to create a reusable collection of files (`excludes.txt`, `patterns.txt`). I could not make it work with `github.action_path`, is there a way to use `github.action_path`? 

I also tried to set the `config` parameter to multiple different folders, and only folders starting with `.github` are working for me. How does this parameter work?

Using `v0.0.20`

**Comment by jsoref** [comment]
I've recently (to be available in a future release) added a [`spell_check_this`]( https://github.com/check-spelling/check-spelling/blob/871e5e232fd1f6fd609cec3f764335dbf93ef0f0/action.yml#L222) parameter which has let me do some things. I'm still experimenting with it.

I tried to make a reusable workflow for this but found it to be a real mess. I hope you're using a split permission model in your attempt. 

Can you provide a link to your workflow? 

You can add `debug: 1` to the workflow to see it work...

**Comment by Luko6** [comment]
Yes, I've tried experimenting with `spell_check_this` without any success, I'm getting an `[unsupported-repo-notation](https://github.com/check-spelling/check-spelling/wiki/Event-descriptions#unsupported-repo-notation)` error, but nothing in the wiki about this.

Links to my workflow:
[action.yml](https://github.com/Lombiq/GitHub-Actions/blob/issue/OSOE-158/.github/actions/check-spelling/action.yml)
[check-spelling.yml](https://github.com/Lombiq/GitHub-Actions/blob/issue/OSOE-158/.github/workflows/check-spelling.yml)

And these trying to be used in another repo:
[workflow](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/blob/6e896370d6af4d798fe57d9fa2dc2000f06d809c/.github/workflows/build-and-test.yml#L26)

Keep in mind these are work in progress and may have commented out code bits around.

**Comment by jsoref** [comment]
Ah, looks like you're using `/` and uppercase letters in your spell-check-this revision. I probably need to add both. Can you temporarily test using a sha instead?

(I've updated the wiki to at least mention the error.)

**Comment by Luko6** [comment]
It's the same with the sha.

[PR](https://github.com/Lombiq/GitHub-Actions/pull/27) I'm testing with.

**Comment by Luko6** [comment]
Great, the error is gone!

**Comment by jsoref** [comment]
Ok, I think I've fixed this in `prerelease` -- Ideally you should be able to use the original mixed-case repository name and the branch name w/ its `/`.

Please let me know if it works.

Note that I don't have a particular eta for releasing the next version (although I do want to release one soonish).

**Comment by Luko6** [comment]
Not really sure if related, but when using `spell_check_this`, is `config` ignored?

**Comment by Luko6** [comment]
I've added `checkout: 1` and get the same.

Testing with [this PR](https://github.com/Lombiq/GitHub-Actions/pull/27)

**Comment by jsoref** [comment]
```
    uses: Lombiq/GitHub-Actions/.github/workflows/check-spelling.yml@issue/OSOE-158
```
Appears to point to:
https://github.com/Lombiq/GitHub-Actions/blob/issue/OSOE-158/.github/workflows/check-spelling.yml

Which in turn has:
```
      - name: Check Spelling
        uses: Lombiq/GitHub-Actions/.github/actions/spelling@issue/OSOE-158
```

But, that doesn't exist, there's no `spelling` in:
https://github.com/Lombiq/GitHub-Actions/tree/issue/OSOE-158/.github/actions

(The maze of twisty indirections in github reusable workflows is really dizzying. I remember trying to make everything work when I tried to write one for this... I'm not sure where mine is hiding, but, I basically gave up when I realized the complexity didn't seem to help much...)

I probably won't be able to respond until late next week. Good luck.

**Comment by jsoref** [comment]
I believe this is fixed with the release of [`v0.0.21`](https://github.com/check-spelling/check-spelling/releases/tag/v0.0.21)

I haven't forgotten about the reusable workflow offer. I suppose I should create a ticket or wiki item for it...
